plugins/UsersManager/UserUpdater.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\UsersManager;
+
+use Piwik\API\Request;
+
+class UserUpdater
+{
+
+    /**
+     * Use this method if you have to update the user without having the ability to ask the user for a password confirmation
+     * @param $userLogin
+     * @param bool $password
+     * @param bool $email
+     * @param bool $alias
+     * @param bool $_isPasswordHashed
+     * @throws \Exception
+     */
+    public function updateUserWithoutCurrentPassword($userLogin, $password = false, $email = false, $alias = false,
+                                                     $_isPasswordHashed = false)
+    {
+        API::$UPDATE_USER_REQUIRE_PASSWORD_CONFIRMATION = false;
```

Comment:
Added this class so each time this is needed it doesn't need to be re-developed and no mistakes can be made. (eg forgetting to reset `API::$UPDATE_USER_REQUIRE_PASSWORD_CONFIRMATION = true;`)